### PR TITLE
Add Swift Package Manager Support

### LINKED ISSUE #172: SPM Support
Did you plan to support Swift Package Manager?

### PR DESCRIPTION
**Pull request by AvdLee**
Finally, we can add SPM support now that resources are supported! 

Besides:
- [x] Optimised folder structure for assets. Localization files where everywhere
- [x] Tests are succeeding again

Fixes #172

### CODE REVIEW

**Review by Boris-Em — APPROVED** [review]

**Comment by wetransferplatform** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  danger-swift` failed.
  </td>
    </tr>
  </tbody>
</table>



### Log


```sh
/var/folders/6q/wgy6jtp12w5gzgm9lzcglpqw0000gn/T/_tmp_dangerfile.swift:1:8: error: no such module 'Danger'
import Danger
       ^
[31mERROR: Dangerfile eval failed at Dangerfile.swift
[0;0m[31mERROR: Could not get the results JSON file at /var/folders/6q/wgy6jtp12w5gzgm9lzcglpqw0000gn/T/danger-response.json
[0;0m
```

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against bd0fe4f6d63254a4d38f1cecb19a20e186752fdf
</p>

File: README.md [hunk 1]
```diff
@@ -49,6 +49,18 @@ It's modelled after `UIImagePickerController`, which makes it a breeze to use.
 
 ## Installation
 
+### Swift Package Manager
+
+The [Swift Package Manager](https://swift.org/package-manager/) is a tool for automating the distribution of Swift code and is integrated into the `swift` compiler. It is in early development, but Alamofire does support its use on supported platforms.
```
Comment (kairadiagne) [inline README.md:54]: Were referencing `Alamofire` instead of `WeScan`.

Comment (AvdLee) [reply README.md:54]: Oops

**Review by kairadiagne — APPROVED** [review]

**Comment by wetransferplatform** [comment]
Congratulations! :tada: This was released as part of [Release 1.8.0](https://github.com/WeTransfer/WeScan/releases/tag/1.8.0) :rocket:<p align="right"><i>Generated by <a href="https://github.com/WeTransfer/GitBuddy">GitBuddy</a></i></p>

**Pull request merged**